suite-native/app/metro.config.js

```diff
@@ -54,6 +56,13 @@ const config = {
                 };
             }
 
+            if (process.env.IS_DETOX_BUILD && moduleName === '@trezor/connect') {
```

Comment:
This looks a bit overcomplicated to me and it is global for all tests. Have you tried `jest.mock`?

Reply:
Aha, I see it is probably not working in Detox, right? https://wix.github.io/Detox/docs/guide/mocking/

Reply:
Yes, it does not :/